install/roles/dovecot/files/access-check-blacklist.sh

```diff
@@ -65,7 +65,7 @@ fi
 # Check if the IP address is blacklisted
 blacklisted="0"
 if [ -r "$blacklistFile" ]; then
-    blacklisted=$(grepcidr -x -c "$IP" "$blacklistFile")
+    blacklisted=$(echo "$IP" | grepcidr -x -c -f "$blacklistFile")
```

Comment:
Maybe keep this change for another commit.

Reply:
Yes, you are right. It is just painful to switch constantly to multiple branches.

Reply:
Removed, it will be in another PR.